### ISSUE TITLE
Missing powerful front-end repositories

### ISSUE THREAD

**Issue by mohamadrezakoohkan** [MRezaSafari/AwesomeFrontend#16]
So far, I have seen things that were really interesting and enjoyable, but there is a need for other front-end repositories to develop both professional and simple projects.
For example [CWA](https://github.com/corona-warn-app!) is an organization which made a lot of good work in github.
https://github.com/corona-warn-app/cwa-website

**Comment by MRezaSafari** [comment]
So! what can we do here?
is it somehow your general concern? or it's related to this repository?

**Comment by mohamadrezakoohkan** [comment]
Yes, It is related to this repo. We can provide some of them in https://github.com/MRezaSafari/AwesomeFrontend/blob/main/topics/full-projects.md section if you agree.

**Comment by MRezaSafari** [comment]
let's move this to discussion board.
